package.json

```diff
@@ -14,53 +14,53 @@
     "prepare": "patch-package"
   },
   "dependencies": {
-    "@ecl/builder": "3.2.2",
-    "@ecl/preset-ec": "3.2.2",
-    "@ecl/preset-eu": "3.2.2",
-    "@ecl/preset-reset": "3.2.2",
+    "@ecl/builder": "3.2.3",
+    "@ecl/preset-ec": "3.2.3",
+    "@ecl/preset-eu": "3.2.3",
+    "@ecl/preset-reset": "3.2.3",
     "@ecl/stylelint-config": "0.2.1",
-    "@ecl/theme-ec": "3.2.2",
-    "@ecl/theme-eu": "3.2.2",
-    "@ecl/twig-component-accordion": "3.2.2",
-    "@ecl/twig-component-blockquote": "3.2.2",
-    "@ecl/twig-component-breadcrumb-core": "3.2.2",
-    "@ecl/twig-component-breadcrumb-standardised": "3.2.2",
-    "@ecl/twig-component-button": "3.2.2",
-    "@ecl/twig-component-card": "3.2.2",
-    "@ecl/twig-component-datepicker": "3.2.2",
-    "@ecl/twig-component-description-list": "3.2.2",
-    "@ecl/twig-component-expandable": "3.2.2",
-    "@ecl/twig-component-fact-figures": "3.2.2",
-    "@ecl/twig-component-file": "3.2.2",
-    "@ecl/twig-component-footer-core": "3.2.2",
-    "@ecl/twig-component-footer-standardised": "3.2.2",
-    "@ecl/twig-component-gallery": "3.2.2",
-    "@ecl/twig-component-hero-banner": "3.2.2",
-    "@ecl/twig-component-icon": "3.2.2",
-    "@ecl/twig-component-inpage-navigation": "3.2.2",
-    "@ecl/twig-component-language-list": "3.2.2",
-    "@ecl/twig-component-link": "3.2.2",
-    "@ecl/twig-component-label": "3.2.2",
-    "@ecl/twig-component-media-container": "3.2.2",
-    "@ecl/twig-component-menu": "3.2.2",
-    "@ecl/twig-component-message": "3.2.2",
-    "@ecl/twig-component-page-banner": "3.2.2",
-    "@ecl/twig-component-page-header-core": "3.2.2",
-    "@ecl/twig-component-page-header-standardised": "3.2.2",
-    "@ecl/twig-component-pagination": "3.2.2",
-    "@ecl/twig-component-radio": "3.2.2",
-    "@ecl/twig-component-search-form": "3.2.2",
-    "@ecl/twig-component-site-header-core": "3.2.2",
-    "@ecl/twig-component-site-header-standardised": "3.2.2",
-    "@ecl/twig-component-skip-link": "3.2.2",
-    "@ecl/twig-component-social-media-follow": "3.2.2",
-    "@ecl/twig-component-social-media-share": "3.2.2",
-    "@ecl/twig-component-table": "3.2.2",
-    "@ecl/twig-component-tag": "3.2.2",
-    "@ecl/twig-component-text-input": "3.2.2",
-    "@ecl/twig-component-timeline": "3.2.2",
-    "@ecl/vanilla-utility-background": "3.2.2",
-    "@ecl/vanilla-utility-border": "3.2.2",
+    "@ecl/theme-ec": "3.2.3",
+    "@ecl/theme-eu": "3.2.3",
+    "@ecl/twig-component-accordion": "3.2.3",
```

Comment:
Should we add a new "category filter" (https://ec.europa.eu/component-library/playground/ec/?path=/story/components-category-filter--default) component?